src/gridstack.ts

```diff
@@ -2035,11 +2037,29 @@ export class GridStack {
       if (!node) return;
 
       helper = helper || el;
+
+      // if the element is being dragged from outside, scale it down to match the grid's scale
+      // and slightly adjust its position relative to the mouse
+      if (!node.grid?.el) {
+        // this scales the helper down
+        helper.style.transform = `scale(${1 / this.dragTransform.xScale},${1 / this.dragTransform.yScale})`;
+        // this makes it so that the helper is well positioned relative to the mouse after scaling
+        const helperRect = helper.getBoundingClientRect();
+        helper.style.left = helperRect.x + (this.dragTransform.xScale - 1) * (event.clientX - helperRect.x) / this.dragTransform.xScale + 'px';

```

Comment:
not sure I understand the "slightly adjust its position" and code here - it s mapping exactly to the same relative point (vido seem that way). can't figure from the code right now

Reply:
Yes, that code is mapping to the same relative position after scaling. the mouse position relative to the element should be the same after scaling

Reply:
@Amdoun @adumesny These lines (setting `left` and `top` on `helper`) doesn't work well when the draggable has different `appendTo` than `body` (defined in `setupDragIn`), and when this `appendTo` element has `top`/`left` greater than 0.

Helper element receives mouse positions + offset of the `appendTo` element, placing it wrong.

Reply:
@creage, please file a bug with a test case showing the issue. I'll have to remind myself why we have `appendTo:element` to start with (likely historical jquery UI option) as I'm not sure the use case - if you can comment on that as well.
supporting too many options makes the lib complicated/briddle.

Reply:
@adumesny Well, we don't want to add draggable helper to the body, but we want them to stick under some parent container - this way we can adjust styling of such helpers easily, and less pollute to the global `body` scope.

Like, a Dialog that shows a GridStack with some items we can drag to the grid. All styling is defined within the Dialog component, which we can use in different areas in our app.

So, this option is quite useful - please don't remove it.

Reply:
| All styling is defined within the Dialog component

ok makes sense. but then aren't items clipped by the dialog when dragging around which is a little odd, even if you can only drop them in the grid of the dialog.